scripts/create_newcase

```diff
@@ -82,10 +82,14 @@ def parse_command_line(args, cimeroot):
                         "with the new user grid changes added to it"
 			"This argument is required if --user-grid is True")
 
-    parser.add_argument("--srcroot",default=os.path.dirname(cimeroot),
+    parser.add_argument("--srcroot", default=os.path.dirname(cimeroot),
                         help="Alternative path for source root directory. By default this is set to"
                         "cimeroot/../")
 
+    if cime_model == "acme":
```

Comment:
@rljacob : this introduces an important problem. What am I supposed to do with features that were implemented poorly on the ACME side? Am I supposed to refactor the work to do it right? Or throw it out and ask the developers to reimplement once ACME is using ESMCI CIME?